### ISSUE TITLE
Repeated Tenor Error when tenor part begins with a ligature

### ISSUE THREAD

**Issue by DanielS146** [MeasuringPolyphony/mp_editor#58]
For some reason, the initial ligature of the first staff of the tenor is being excised from the repeat when transcribed into Score Editor.

[Super cathedram-Presidentes Console Log.log](https://github.com/MeasuringPolyphony/mp_editor/files/5071989/Super.cathedram-Presidentes.Console.Log.log)
[Super cathedram-Presidentes MEI Parts and Score.zip](https://github.com/MeasuringPolyphony/mp_editor/files/5071990/Super.cathedram-Presidentes.MEI.Parts.and.Score.zip)

![Tenor Repeat Error Edit](https://user-images.githubusercontent.com/69400378/90197615-234bde00-dd9d-11ea-9898-0efd4886c8f7.jpg)

**Comment by martha-thomae** [comment]
Can this be repeated in the new version?

**Comment by karend27** [comment]
When I attempted to repeat this in the new version I got an error message when I moved to Score editor. I entered a tenor line that begins with a ligature, set the end of the repeated tenor, entered "4" as the number of repeats, and clicked continue to score editor.

I'm pasting the screenshot of the error and the saved MEI parts file and console.
[Archive 3.zip](https://github.com/MeasuringPolyphony/mp_editor/files/5174962/Archive.3.zip)
:
<img width="909" alt="Screen Shot 2020-09-04 at 9 02 21 AM" src="https://user-images.githubusercontent.com/11195203/92242603-08e1bd80-ee8e-11ea-8373-78c34fd42554.png">
